### PR TITLE
📖 doc: update feature list

### PR DESCRIPTION
**Pull request by cosmic-flood**
1. Add workflow method in the feature list
2. Add SSO in the feature list
3. Add Helm Charts installation in the feature list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,10 +134,12 @@ FeatBit offers a range of features including:
 
 - **[Audit Log](https://docs.featbit.co/docs/feature-flags/audit-log)**: Keep track of feature flag and segment changes.
 
-- **[Feature Workflow](https://docs.featbit.co/docs/feature-flags/feature-workflow)**: Control your use of feature flags by creating complex automated workflows within FeatBit.
+- **[Feature Workflow](https://docs.featbit.co/docs/feature-flags/feature-workflow)**: Control your use of feature flags by creating complex automated workflows within FeatBit (Flag Triggers, Schedulers, Change Approve).
```
Comment (deleteLater) [inline README.md:138]: Flag Triggers, Scheduled Flag Changes, Change Approve Requests

File: README.md [hunk 2]
```diff
@@ -134,10 +134,12 @@ FeatBit offers a range of features including:
...
 
 - **[Web APIs](https://docs.featbit.co/docs/integrations/using-featbit-rest-api)**, automate your workflow with Web APIs.
 
+- **SSO**, integrate with your existing SSO provider.
```
Comment (deleteLater) [inline README.md]: Integrate with your existing Identity Provider

File: README.md [hunk 3]
```diff
@@ -148,6 +150,8 @@ FeatBit offers a range of features including:
 
 - **[ChatGPT Tech Debt Reduction](https://github.com/featbit/featbit/blob/doc/readme/llm/README.md)** (experimental features): Utilize ChatGPT4 and FeatBit's VSCode extension to minimize technical debt associated with feature flagging.
 
+- **[Helm Charts Installation](https://github.com/featbit/featbit-charts/)**, FeatBit can be installed on-premises, in the cloud, or in a hybrid environment trhough Helm Charts.
```
Comment (deleteLater) [inline README.md]: typo: **through** Helm Charts.

Comment (cosmic-flood) [reply README.md]: some times, chatgpt makes mistake too lool

**Comment by cosmic-flood** [comment]
@deleteLater comments resolved

**Review by deleteLater — APPROVED** [review]
LGTM

**Pull request merged**